js/apps/admin-ui/src/dashboard/Dashboard.tsx

```diff
@@ -138,11 +149,79 @@ const Dashboard = () => {
           data-testid="dashboard-tabs"
           defaultLocation={toDashboard({
             realm,
-            tab: "info",
+            tab: "welcome",
           })}
           isBox
           mountOnEnter
         >
+          <Tab
+            id="welcome"
+            data-testid="welcomeTab"
+            title={<TabTitleText>{t("welcomeTabTitle")}</TabTitleText>}
+            {...welcomeTab}
+          >
+            <PageSection variant="light">
+              <div className="pf-l-grid pf-u-ml-lg">
+                <div className="pf-l-grid__item pf-m-12-col">
+                  <Title
+                    className="pf-u-font-weight-bold"
+                    headingLevel="h2"
+                    size="3xl"
+                  >
+                    {t("welcome")} {realmDisplayInfo}
+                  </Title>
+                </div>
+                <div className="pf-l-grid__item pf-m-12-col pf-u-mt-md">
+                  <Text component={TextVariants.h3}>
+                    {t("welcomeDescriptionA")}
+                  </Text>
+                  <Text className="pf-u-mt-xs" component={TextVariants.h3}>
+                    {t("welcomeDescriptionB")}
```

Comment:
Could we place this text in a single label instead? I believe we can add things like line breaks there.